Source/WebKit/WebProcess/GPU/graphics/RemoteRenderingBackendProxy.cpp

```diff
@@ -121,27 +121,34 @@ void RemoteRenderingBackendProxy::ensureGPUProcessConnection()
 template<typename T, typename U, typename V, typename W>
 auto RemoteRenderingBackendProxy::send(T&& message, ObjectIdentifierGeneric<U, V, W> destination)
 {
-    auto result = streamConnection().send(std::forward<T>(message), destination, defaultTimeout);
+    RefPtr connection = this->connection();
+    if (UNLIKELY(!connection))
+        return IPC::Error::Timeout;
```

Comment:
Is `Timeout` the right error here? Would `InvalidConnection` be more accurate (like you use in `sendSync`)?